src/components/items/productId/ProductSummary.jsx

```diff
@@ -0,0 +1,49 @@
+import styles from './ProductSummary.module.css';
+import { useState } from 'react';
+import { ReactComponent as EmptyHeart } from 'assets/imgs/empty_heart.svg';
+import { ReactComponent as FillHeart } from 'assets/imgs/fill_heart.svg';
+
+const ProductSummary = ({
+  productName, 
+  productPrice, 
+  productDescription, 
+  productTags,
+  productFavoriteCount,
+  ownerNickname,
+  ownerImage,
+  updateAt,
+}) => {
+  const [isEmptyHeart, setIsEmptyHeart] = useState(true);
+
+  const onClick = () => {
+    setIsEmptyHeart(!isEmptyHeart);
+  }
+
+  return (
+    <div className={styles['container']}>
+      <h4 className={styles['product-name']}>{productName}</h4>
+      <p className={styles['product-price']}>{Number(productPrice).toLocaleString()}</p>
+      <hr className={styles['line']} />
+      <p className={styles['product-description-title']}>상품 소개</p>
+      <p className={styles['product-description']}>{productDescription}</p>
+      <p className={styles['product-tag-title']}>상품 태그</p>
+      <div className={styles['product-tag-container']}>
+        {productTags.map((tag, index) => (
+          <span key={index}className={styles['product-tag']}>#{tag}</span>
+        ))}
+      </div>
+      {productTags.length === 0 && <span className={styles['empty-tag']}></span>}
```

Comment:
P3:
아래 코드도 동일동작합니다.

```suggestion
      {!productTags.length && <span className={styles['empty-tag']}></span>}
```